test/integration/connection-monitoring-and-pooling/connection_monitoring_and_pooling.spec.test.ts

```diff
@@ -52,3 +53,22 @@ describe('Connection Monitoring and Pooling Spec Tests (Integration)', function
     )
   });
 });
+
+describe('Connection Monitoring and Pooling Spec Tests (Integration) - logging', function () {
+  const tests = loadSpecTests('connection-monitoring-and-pooling', 'logging');
+
+  runUnifiedSuite(tests, (test: { description: string }) => {
```

Comment:
```suggestion
  runUnifiedSuite(tests, test => {
```

No need to annotate something that has type info already